manage

```diff
@@ -214,7 +214,15 @@ EOF
 
 #An apt repository's Packages file can be corrupted so that an apt update will silently fail. See: https://bugs.launchpad.net/ubuntu/+source/apt/+bug/1809174
 #This line will fix the problem by removing any zero-size Packages files.
-find /var/lib/apt/lists -type f -name '*Packages' -size 0 2>/dev/null | xargs sudo rm -f
+removal_list="$(find /var/lib/apt/lists -type f -name '*Packages' -size 0 2>/dev/null)"
+if [ ! -z "$removal_list" ]; then
```

Comment:
In nearly all cases, the removal list is going to be empty. If your intention is to avoid typing in the sudo password more than once, the dialog should be launched whether or not any Packages files are empty. I would advise putting this in the `terminal_manage` function in the gui script.

Reply:
no this is not my intention. I'm fine with typing in the sudo password once in the terminal.
in my experience, the reason the other prompts were showing is BECAUSE this prompt was running sudo every time.

now that it doesn't run unless necessary, the other windows don't show up, and you only type in the sudo to the terminal once. I've tested every possible combination and only one terminal shows up now, and you only type in the password once

Reply:
basically, the check from here was added: https://github.com/Botspot/pi-apps/pull/1580#issuecomment-1107918726
and then to deal with the common edge case that the manage script is running in a hidden terminal (which it usually is) the yad dialog is used to get the users sudo password

Reply:
I guess that's fine, but I consider it quite strange to have a dedicated gui for an edge-edge-case scenario. I say "edge" twice, because not only does a Packages file need to be corrupted, but also `sudo` needs to require a password, which is not the default on RPiOS.
So basically, this password GUI may very well not ever be shown, to a single user, for the lifetime of Pi-Apps.